lib/jobs/obm-control.js

```diff
@@ -64,18 +64,33 @@ function obmControlJobFactory(
             assert.ok(node, "Node should exist to run OBM command");
             assert.ok(node.obmSettings, "Node should have OBM settings to run OBM command");
 
-            var settings = _.find(node.obmSettings, function(setting) {
+            //var settings = _.find(node.obmSettings, function(setting) {
+            self.settings = _.find(node.obmSettings, function(setting) {
                 return setting ? setting.service === self.obmServiceName : false;
             });
 
-            assert.ok(settings, "Node should have OBM settings for service: " +
+            if(!self.settings){
```

Comment:
I think I follow you here, thanks for the feedback! Just curious, could you help me understand what the case would be that we don't want the default behavior?